docs/core/deploying/native-aot/diagnostics.md

```diff
@@ -78,3 +78,7 @@ Platform-specific tools like [PerfView](https://github.com/microsoft/perfview) a
 ## Heap analysis
 
 Managed heap analysis isn't currently supported in Native AOT. Heap analysis tools like [dotnet-gcdump](../../diagnostics/dotnet-gcdump.md), [PerfView](https://github.com/microsoft/perfview), and Visual Studio heap analysis tools don't work in Native AOT in .NET 8.
+
+## See also
+
+- [Native AOT diagnostic support on iOS and Mac Catalyst](https://learn.microsoft.com/dotnet/maui/deployment/nativeaot?view=net-maui-9.0&preserve-view=true#native-aot-diagnostic-support-on-ios-and-mac-catalyst)
```

Comment:
```suggestion
- [Native AOT diagnostic support on iOS and Mac Catalyst](/dotnet/maui/deployment/nativeaot#native-aot-diagnostic-support-on-ios-and-mac-catalyst)
```

Reply:
@gewarren the URL needs to point to the .NET 9 version of the documentation. Not sure how this would work with relative links?

Reply:
It's better not to pin it to a version, so when the next version comes out, it's not pinned to an old version. Also, I tried out the link I suggested, and it defaults to .NET 9, so I think it's fine as suggested.

https://learn.microsoft.com/dotnet/maui/deployment/nativeaot#native-aot-diagnostic-support-on-ios-and-mac-catalyst

Reply:
Thanks for the clarification, I added your suggestion. 
I think we are now good to merge this?